partcad-cli/src/partcad_cli/click/commands/list/packages.py

```
@@ -0,0 +1,33 @@
+    with Process("List", "this"):
+        # TODO: Show source (URL, PATH) of the package, probably use prettytable as well
+        pkg_count = 0
+
+        projects = ctx.get_all_packages()
+        projects = sorted(projects, key=lambda p: p["name"])
+
+        output = "PartCAD packages:\n"
+        for project in projects:
+            project_name = project["name"]
+
+            line = "\t%s" % project_name
```

Ruff (0.8.0)

Use format specifiers instead of percent format

Replace with format specifiers

(UP031)